media-sound/teamspeak-client/teamspeak-client-3.2.3-r2.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop unpacker xdg-utils toolchain-funcs
```

Comment:
Sorry, missted that. Alphabetical order please.

Reply:
Done.